src/localplayer.cpp

```diff
@@ -151,9 +152,318 @@ static bool detectSneakLadder(Map *map, INodeDefManager *nodemgr, v3s16 pos)
 
 #undef GETNODE
 
+// Move code from before new sneak.
+// Enabled by the 'classic_sneak' option (disabled by default).
+void LocalPlayer::classic_move(f32 dtime, Environment *env, f32 pos_max_d,
+		std::vector<CollisionInfo> *collision_info)
+{
```

Comment:
the code duplication here is very horrible

also you will need a note that any changes to the movement code need to be done in both functions